src/Documentation/Markdown/lang/dvc.js

```diff
@@ -34,37 +34,37 @@ let _javascript = function(hljs) {
     aliases: ['dvc'],
     contains: [
       {
-        begin: /^\s*\$/,
-        end: /([^\\]\n)|\Z/,
+        begin: /    \$/,
+        className: 'skipped'
+      },
+      {
+        begin: /\bls\b|\bcat\b|\bvi\b|\bmkdir\b|\bcd\b|\bwget\b|\bdu\b|\bpython\b|\bcp\b|\bexport\b|\becho\b|\bpip\b|\bcurl\b|\btar\b|\bexec\b|\bautoload\b|\bsudo\b|\bunzip\b|\brm\b|\btree\b|\bfile\b|\bmd5\b|\bsource\b|\bvirtualenv\b/,
```

Comment:
why do we need this? can we actually make `className: 'skipped'` just a sub-contains as `srtrong` and others were before?

Reply:
The documentation for "react-syntax-highlighter" does not contain instructions for customizing dictionaries. The rules can replace each other, on what basis - is unknown. I tried several options, but only this one works correctly.